### PR TITLE
fix build error in debian 10

### PR DESCRIPTION
**Pull request by ruansteve**
## Types of changes

 
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [ ] Other (please describe):

## Checklist:

- [ ] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [ ] I have linked this PR to one or more Phabricator Task(s)
- [ ] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

**Comment by ruansteve** [comment]
shixin

File: data/defaults.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "architecture": "amd64",
-  "debian_mirror": "http://deb.debian.org/debian",
-  "debian_security_mirror": "http://deb.debian.org/debian-security",
+  "debian_mirror": "https://mirrors.ustc.edu.cn/debian/",
```
Comment (dmbaturin) [inline data/defaults.json:3]: What is the motivation for this change? Using a specific third-party mirror, not affiliated with the Debian project, needs really strong motivation.

File: data/live-build-config/hooks/live/40-linux-firmware.chroot [hunk 2]
```diff
@@ -6,15 +6,16 @@ COMMIT="aa95e90b2c638f1ca6647d12b2d1b18284428f9c"
...
-    exit 1
-fi
+#if [ "${ret}" != "200" ]; then
+#    echo E: Received HTTP error code \"${ret}\" when downloading Linux Firmware ...
+#    exit 1
+#fi
 
-tar xf /tmp/linux-firmware-${COMMIT}.tar.gz -C /tmp
+#tar xf /tmp/linux-firmware-${COMMIT}.tar.gz -C /tmp
+tar xf /home/shixin/linux-firmware-${COMMIT}.tar.gz -C /tmp
```
Comment (dmbaturin) [inline data/live-build-config/hooks/live/40-linux-firmware.chroot:18]: Most build hosts aren't going to have a user named shixin.

File: data/live-build-config/hooks/live/40-linux-firmware.chroot [hunk 3]
```diff
@@ -6,15 +6,16 @@ COMMIT="aa95e90b2c638f1ca6647d12b2d1b18284428f9c"
 
 echo "I: Retrieving Linux Firmware - commit ${COMMIT}"
 
-ret=$(curl -w "%{http_code}" -o /tmp/linux-firmware-${COMMIT}.tar.gz \
-    https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git/snapshot/linux-firmware-${COMMIT}.tar.gz)
+#ret=$(curl -w "%{http_code}" -o /tmp/linux-firmware-${COMMIT}.tar.gz \
```
Comment (dmbaturin) [inline data/live-build-config/hooks/live/40-linux-firmware.chroot:9]: WTF?

**Review by dmbaturin — CHANGES_REQUESTED** [review]
VyOS 1.2.x is based on Debian Jessie and should be built on Debian Jessie. Building it on a Debian version it's not based on is not supported. We can only merge a fix if it doesn't interfere with building on Jessie and doesn't break compatibility with existing builds.

Yours breaks both. We cannot accept it.
